src/core/annotation.js

```diff
@@ -3433,6 +3435,10 @@ class TextAnnotation extends MarkupAnnotation {
 
     super(params);
 
+    // No rotation for Text (see 12.5.6.4).
+    this.noRotate = true;
+    this.data.hasOwnCanvas = true;
```

Comment:
For overall consistency, given all classes below, how about the following here as well?
```suggestion
    this.data.hasOwnCanvas = this.data.noRotate;
```